src/main/java/gr8pefish/ironbackpacks/block/BlockBackpack.java

```diff
@@ -0,0 +1,255 @@
+        return false;
+    }
+
+    @Override
+    @Deprecated
+    public boolean isOpaqueCube(IBlockState state) {
+        return false;
+    }
+
+    @Override
+    @Deprecated
+    public boolean causesSuffocation(IBlockState state) {
+        return false;
+    }
+
+    @Override
+    @Deprecated
+    public boolean isFullBlock(IBlockState state) {
+        return false;
+    }
+
+    @Override
+    @Deprecated
+    public boolean isFullCube(IBlockState state) {
+        return false;
+    }
+
+    //Add bounding box, with sidedness depending on placement
+
+    @Override
+    public AxisAlignedBB getBoundingBox(IBlockState state, IBlockAccess source, BlockPos pos) {
+        switch (state.getValue(FACING)) {
+            case NORTH:
+                return new AxisAlignedBB(0.9D, 0.0D, 0.235D, 0.1, 1.0D, 0.635D); //x = length, y = height, z = width
+            case EAST:
+                return new AxisAlignedBB(0.36D, 0.0D, 0.1D, 0.76, 1.0D, 0.9D); //x = width, y = height, z = length
+            case SOUTH:
+                return new AxisAlignedBB(0.9D, 0.0D, 0.76D, 0.1, 1.0D, 0.36D); //x = length, y = height, z = width
+            case WEST:
+                return new AxisAlignedBB(0.635D, 0.0D, 0.1D, 0.235, 1.0D, 0.9D); //x = width, y = height, z = length
+        }
+
+        //North default (should be unreachable)
+        return new AxisAlignedBB(0.635D, 0.0D, 0.1D, 0.235, 1.0D, 0.9D); //x1 = width end, x2 = width start, y2= height
+
+    }
+
+    //VARIANT block state
+
+    //TODO: Test, as removed BACKPACK_VARIANT IProperty and haven't tested this updated code yet
+    @Override
+    protected BlockStateContainer createBlockState() {
+        return new BlockStateContainer(this, FACING);
+    }
+
+    @Override
+    public IBlockState getStateFromMeta(int meta) {
+        EnumFacing facing = EnumFacing.Plane.HORIZONTAL.facings()[meta & 0b11]; //first 2 bits for facing, 4 possibilities
+        return getDefaultState().withProperty(FACING, facing);
+    }
+
+    @Override
+    public int getMetaFromState(IBlockState state) {
+        int facing = state.getValue(FACING).ordinal() - EnumFacing.Plane.VERTICAL.facings().length; //0-3, ignore Up and Down
+        return facing; //first 2 bits for facing
+    }
+
+
+    // Other
+
+
+    //TODO: Test
+    @Override
+    public void getDrops(NonNullList<ItemStack> drops, IBlockAccess world, BlockPos pos, IBlockState state, int fortune) {
+
+        //create new itemstack with all the info
+        ItemStack pack = IronBackpacksAPI.applyPackInfo(new ItemStack(IronBackpacksAPI.BACKPACK_ITEM), BackpackInfo.fromTileEntity(this.getTileEntity(world, pos)));
+        drops.add(pack);
+
+        super.getDrops(drops, world, pos, state, fortune);
+    }
+
+
+
+    //ToDo: Go through placement logic and determine what is necessary
+
+
+    //stolen from vanilla furnace
+    @Override
+    public void onBlockAdded(World worldIn, BlockPos pos, IBlockState state) {
```

Comment:
Stop this cancer. Use `getStateForPlacement(...)` and base it on the placer's horizontal orientation.